tools/infer/utility.py

```diff
@@ -609,8 +610,31 @@ def get_rotate_crop_image(img, points):
     img_crop = img[top:bottom, left:right, :].copy()
     points[:, 0] = points[:, 0] - left
     points[:, 1] = points[:, 1] - top
-    '''
-    assert len(points) == 4, "shape of points must be 4*2"
```

Comment:
Situations where more data formats need to be processed in addition to CTW

Reply:
Yes. But both in CTW, ICDAR2015 and our own annotation files we don't assume bbox consists of fixed number of points to form a polyline.

So we just treat it as general polyline: Not 2 points (upper left, bottom right), 3 points (trianguulars), polygons